patches/analysisCopyNumberEstimation.xml

```diff
@@ -128,7 +128,7 @@
 
         <!--estimatePurityPloidy Opts-->
         <cvalue name='min_length_purity' value='1000000' type="integer"/>
-        <cvalue name='min_hetSNPs_purity' value='500' type="integer"/>
+        <cvalue name='min_hetSNPs_purity' value='50' type="integer"/>
```

Comment:
Just a remark: AFAICS this will change the results. Carefully consider, whether setting this value _ad hoc_ for a specific analysis would not be a better solution as changing the output essentially means breaking the compatibility with older results. Are there official ICGC guidelines for how to handle compatibility-breaking changes?

Reply:
@vinjana, appreciate that you pointed this out. Our goal at this moment is to preserve compatibility to enable reproducibility of PCAWG variant callings. As such, we can not accept this change.

In long run, we have the idea to expose tuning parameters like this to endusers, so that it's possible to customize at workflow invocation time.

Reply:
@ZeepXanflorp, can you please create a separate issue for that? I had similar experience (not limited to this workflow) as well. I think the general idea here is that lack of result should not crash the workflow run.